vue/test/components/steps/steps.test.js

```diff
@@ -0,0 +1,75 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Steps", () => {
+    it("should initialize the steps", () => {
+        const component = base.getComponent("Steps", {
+            props: { steps: ["A", "B", "C"], step: 1 }
+        });
+        assert.strictEqual(component.exists(), true);
+    });
+    it("should only highlight the current step", () => {

```

Comment:
Newline between `it`s I think. Did you get this (they being together) from any place?